tests/app/email_branding/test_rest.py

```diff
@@ -149,6 +178,34 @@ def test_post_update_email_branding_updates_field(admin_request, notify_db_sessi
     assert email_branding[0].text == email_branding[0].name
 
 
+@freezegun.freeze_time()
+def test_post_update_email_branding_updated_fields(admin_request, notify_db_session, sample_user):
+    data = {"name": "test email_branding", "logo": "images/text_x2.png"}
+    response = admin_request.post("email_branding.create_email_branding", _data=data, _expected_status=201)
+
+    email_brandings = EmailBranding.query.all()
+    email_branding = email_brandings[0]
+    assert len(email_brandings) == 1
+    assert email_branding.updated_by is None
+    assert email_branding.updated_at is None
+
+    email_branding_id = response["data"]["id"]
+    update_data = {"name": "updated email branding name", "updated_by": str(sample_user.id)}
+
+    admin_request.post(
+        "email_branding.update_email_branding",
+        _data=update_data,
+        email_branding_id=email_branding_id,
+    )
+
+    email_brandings = EmailBranding.query.all()
+    email_branding = email_brandings[0]
+    assert len(email_brandings) == 1
+    assert str(email_branding.id) == email_branding_id
+    assert str(email_branding.updated_by) == update_data["updated_by"]
+    assert email_branding.updated_at == datetime.datetime.utcnow()
```

Comment:
should we maybe set a different time for create and update operations, and then assert that created_at was not updated when doing the update operation?

Reply:
Good call - pushed a commit adding this check :)